Cargo.toml

```diff
@@ -12,6 +12,8 @@ name = "secret-share-combine"
 path = "src/bin/combine.rs"
 
 [dependencies]
+log = "0.3.8"
+env_logger = "0.4.3"
```

Comment:
I think I would like to support all the patch versions of these crates. Could this be changed to the following?

```
log = "0.3"
env_logger = "0.4"
```